BetterResults.js

```diff
@@ -192,6 +194,34 @@ function onLoad() {
   transition: visibility 0s, opacity 0.5s linear;
 }
 
+#details {
+  position: relative;
+}
+
+#details:hover::before {
+  visibility: visible;
+  opacity: 0.9;
+}
+
+#details::before {
+  content: attr(content);
+  color: white;
+  background-color: #1a1a1a;
+  margin-left: -60px;
+  visibility: hidden;
+  position: absolute;
+  text-align: center;
+  padding: 5px 0;
+  border-radius: 6px;
+  z-index: 1;
+  opacity: 0;
+  transition: opacity .6s;
+  width: 240px;
+  top: 100%;
+  left: 50%;
+  margin-left: -120px;
+}
```

Comment:
Can you change these to `rem` instead of `px` units. You also left a second margin-left at the top.